splash/qtrender_lua.py

```diff
@@ -487,6 +491,75 @@ def error(error_info):
         ))
         return cmd
 
+    @command(async=True, sets_callback=True, decode_arguments=False)
```

Comment:
hm, does sets_callback really work here? I recall it requires the first argument to be a callback